Whiskey/Tasks/Set-WhiskeyProGetAsset.ps1

```diff
@@ -0,0 +1,97 @@
+
+
+function Set-WhiskeyProGetAsset {
+    <#
+        .SYNOPSIS
+        Adds and Updates assets to the ProGet asset manager. 
+
+        .DESCRIPTION
+        The `Set-WhiskeyProGetAsset` adds assets to ProGet. A root directory needs to be 
+        created in ProGet using the `New-ProGetFeed` function in ProGetAutomation with Type `Asset`.
+        
+        * `Path` (Mandatory): The relative paths to the files/directories to upload to ProGet. Paths should be relative to the whiskey.yml file they were taken from.
+        * `Uri` (Mandatory): The uri to the ProGet instance.
+        * `Name` (Mandatory): The desired Name you wish the file to be named in ProGet.
+        * `Directory` (Mandatory): The Path to the Directory you wish to upload the asset to.
+        * `ApiKey` (Mandatory): The ApiKey to Proget Api.
+        * `ProGetUsername` (Mandatory): The Username to the ProGet Api.
+        * `ProGetPassword` (Mandatory): The Password to the ProGet Api.
+
+        .EXAMPLE       
+        BuildTasks:
+        - SetProGetAsset:
+            Path: 'path/to/file.txt'
+            Uri: http://proget.dev.webmd.com/
+            Name: 'exampleAsset'
+            Directory: 'versions'
+            ApiKey: 'exampleApiKey'

```

Comment:
Credentials and API keys don't get passed to a task. Instead, you use ApiKeyID and CredentialID properties that reference credentials the user has added via the `Add-WhiskeyCredential` function. See the `PublishProGetUniversalPackage` task.

Reply:
Done